test/tools/route_tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+PATH_BIN="../../bazel-bin/test/tools/router_check/router_check_tool"
+PATH_CONFIG="config"
+
+# Testing expected matches
+eval "$PATH_BIN" "$PATH_CONFIG/TestRoutes_router.json" "$PATH_CONFIG/TestRoutes_expected.json" "--details"
+eval "$PATH_BIN" "$PATH_CONFIG/ContentType_router.json" "$PATH_CONFIG/ContentType_expected.json"
+eval "$PATH_BIN" "$PATH_CONFIG/HeaderMatchedRouting_router.json" "$PATH_CONFIG/HeaderMatchedRouting_expected.json"
```

Comment:
You should be able to do this less verbosely with a for loop. E.g.

TESTS="ContentType HeaderMatchedRouting ..."
for t in "$TESTS"
do
  "$PATH_BIN" "$PATH_CONFIG/$t_router.json" "$PATH_CONFIG/$t_expected.json
done